### PR TITLE
Beakers, bottles and syringe resprite

### PR DESCRIPTION
**Pull request by alsoandanswer**
Mostly ported from Eris, noreact beaker and bottle variants by yours truly. Also, inhands, again. Because by jove, I made this system, and I'm gonna use the shit out of it.

Removed some unused sprites as well, and some ancient botany garbage.

Rather surprised at the reception this got, apparently people really like to have beakers that they don't have to pixel-hunt to click. Same with syringes.

Also tweaks how the reagent container overlays are handled, shouldn't be too much of a fuss.

edit : also adds new sprites for fire extinguishers and their refill canisters

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!review

**Comment by desvenlafaxine** [comment]
Sprite preview won't load, do you have pics? (Think I know how they look but still).

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/cooking/machinery/cooking_machines/container.dm [hunk 1]
```diff
@@ -307,9 +307,19 @@
...
 	icon_state = "mixingbowl"
+	filling_states = "-10;10;25;50;75;80;100"
 	center_of_mass = list("x" = 17,"y" = 7)
 	max_space = 30
 	matter = list(DEFAULT_WALL_MATERIAL = 300)
 	volume = 180
 	amount_per_transfer_from_this = 10
 	possible_transfer_amounts = list(5,10,15,25,30,60,180)
+
+/obj/item/reagent_containers/cooking_container/plate/bowl/update_icon()
```
Comment (Casper3667) [inline code/modules/cooking/machinery/cooking_machines/container.dm:318]: This isn't called when the bowl reagents are updated.

**Review by Casper3667 — CHANGES_REQUESTED** [review]

**Review by Casper3667 — APPROVED** [review]

**Review by Arrow768 — DISMISSED** [review]
I am concerned about the large syringes.
It was a very common tactic by medical to rush antags with syringes and stab them on harm intent.

The current implementation of the harm-intent-stabbing does not account for armor.
It should be changed to account for armor and make it impossible to harm-intent stab someone with armor. (i.e. "the needle breaks off")
In addition the flat -5 reduction in volume here should be changed to a /2 reduction.
`var/syringestab_amount_transferred = rand(0, (reagents.total_volume - 5))`

**Pull request merged**